packages/plugin-doc-coverage/src/lib/runner/utils.ts

```diff
@@ -0,0 +1,80 @@
+import { SyntaxKind } from 'ts-morph';
+import type {
+  CoverageResult,
+  CoverageType,
+  UnprocessedCoverageResult,
+} from './models.js';
+
+/**
+ * Creates an empty unprocessed coverage report.
```

Comment:
I would call it coverage data and the shape inc the coverage report

Reply:
Love it